.github/workflows/release-test.yml

```diff
@@ -0,0 +1,126 @@
+name: Release-test
+on: [pull_request]
+# use pull request for test, will change to push before merge
+  #push:
+  #  branches:
+  #    - '*'
+  #    - '!master'
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - ubuntu-18.04
+        os_name:
+          - linux_openresty
+          - linux_openresty_1_17
+          - linux_openresty_1_15
```

Comment:
Seems we only need the linux_openresty_xxx plus a test to run cli installed via package?